src/Files.App/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -156,49 +147,39 @@ private async void EjectDevice_Click(object sender, RoutedEventArgs e)
 		private async void OpenInNewTab_Click(object sender, RoutedEventArgs e)
 		{
 			var item = ((MenuFlyoutItem)sender).DataContext as DriveItem;
-			if (await CheckEmptyDrive(item.Path))
-			{
+			if (await DriveHelpers.CheckEmptyDrive(item?.Path))
 				return;
-			}
-			await NavigationHelpers.OpenPathInNewTab(item.Path);
+			await NavigationHelpers.OpenPathInNewTab(item?.Path);
 		}
 
 		private async void OpenInNewWindow_Click(object sender, RoutedEventArgs e)
 		{
 			var item = ((MenuFlyoutItem)sender).DataContext as DriveItem;
-			if (await CheckEmptyDrive(item.Path))
-			{
+			if (item is null || await DriveHelpers.CheckEmptyDrive(item.Path))
```

Comment:
`if (await DriveHelpers.CheckEmptyDrive(item?.Path))` to be in line with the other tests around.

Reply:
Done. Note that I had to edit `NavigationHelpers.OpenPathInNewWindowAsync()`